_includes/plugins/comments_common_issues.md

```diff
@@ -0,0 +1,19 @@
+## Configuring the commented text properties
+
+The highlight styles are now a part of the overall content skin and are changed through customizing the skin.
+
+{{site.productname}} open source project [oxide](https://github.com/tinymce/oxide/blob/master/src/less/theme/content/comments/comments.less) (default skin), defines the variables used for changing the annotation colours.
+
+Refer to the [documentation]({{site.baseurl}}/advanced/creating-a-skin/#creatingaskin) for building a skin using this repo.
+
+For more information on configuring {{site.productname}} formats, refer to the [formats]({{site.baseurl}}/configure/content-formatting/#formats) section.
+
+## Using Comments embedded mode with the Full Page plugin
```

Comment:
This is very confusing to see listed on a page about callbacks. I'd suggest moving this to the embedded page itself, as it makes little sense to show on the callbacks page.

Reply:
Dang. didn't look that closely at this stuff.

Reply:
Fixed in 658e875